samples/started-guides/README.md

```diff
@@ -0,0 +1,29 @@
+# Indy SDK getting started guide
+The following are scripts using the SDK and the Python wrapper. 
```

Comment:
also i am not sure that it is a right place to put this files.

Reply:
@vimmerru I have a different idea about where to put this. I'll talk to you.